spring-xd-module/src/main/java/org/springframework/xd/module/options/DefaultModuleOptionsMetadataResolver.java

```diff
@@ -101,15 +101,7 @@
 
 	private ResourcePatternResolver resourceLoader = new PathMatchingResourcePatternResolver();
 
-	/**
-	 * Construct a new {@link DefaultModuleOptionsMetadataResolver} that will use the provided conversion service when
-	 * converting from String to rich object (supported for {@link PojoModuleOptionsMetadata} only).
-	 *
-	 * @param conversionService Must not be null
-	 */
-	public DefaultModuleOptionsMetadataResolver(ConversionService conversionService) {
-		Assert.notNull(conversionService, "The provided conversionService must not be null.");
-	}
+	private boolean useParentClassLoader = false;
```

Comment:
Yeah, I agree. What do you think about having a flag named `hasModuleClassLoader` and set to `false` here in `DefaultModuleOptionsMetadataResolver` ? This way the module which has its classloader set already would set to `true`.